ftw/crawler/main.py

```diff
@@ -54,77 +55,102 @@ def get_indexing_time(url, indexed_docs, config):
 def crawl_and_index(tempdir, config, options):
     solr = SolrConnector(config.solr)
 
+    try:
+        from slack import SlackLogger
+        SlackLogger = SlackLogger(options.slacktoken)
+        HAS_SLACK = True
+    except ImportError:
+        log.warning('Slacker not found')
```

Comment:
This shouldn't be a warning. At most `INFO` level, and I'd also phrase it differently in order to not suggest that Slacker is required in any way: `Not logging errors to slack (slacker not found)`